### PR TITLE
Update manage-cluster-horizontal-scaling.md

### PR DESCRIPTION
**Pull request by Gabilehner**
changes in logic

### CODE REVIEW

**Comment by PRMerger9** [comment]
@Gabilehner : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/manage-cluster-horizontal-scaling.md [hunk 1]
```diff
@@ -73,7 +71,7 @@ When your cluster approaches a state of under-utilization, scale in to lower cos
     * Number of failed queries are below a defined minimum.
 
 > [!NOTE]
-> The scale in logic currently requires a 7-day evaluation before implementation of optimized scale in. This evaluation takes place once every 24 hours. If a quick change is needed, use [manual scale](#manual-scale).
+> The scale in logic currently requires a 1-day evaluation before implementation of optimized scale in. This evaluation takes place once every 6 hours. If a quicker change is needed, use [manual scale](#manual-scale).
```
Comment (orspod) [inline data-explorer/manage-cluster-horizontal-scaling.md:74]:
```suggestion
> The scale in logic currently requires a 1-day evaluation before implementation of optimized scale in. This evaluation takes place once every 6 hours. If an immediate change is needed, use [manual scale](#manual-scale).
```

**Comment by PRMerger10** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by orspod** [comment]
@Gabilehner - I reviewed your change and committed additional changes. Merging now.
Please make sure you make changes in dataexplorer-docs-pr. Follow email that I sent for more info.

**Pull request merged**